embedx/config.schema.json

```diff
@@ -1371,6 +1371,35 @@
       ],
       "additionalProperties": false
     },
+    "saml": {
+      "type": "object",
+      "properties": {
+        "public_cert_path": {
+          "title": "Path Of Public Certificate",
+          "description": "The local path of the public certificate of Kratos",
+          "type": "string",
+          "format": "uri"
```

Comment:
We usually support fetching from various sources:

- base64
- http(s)
- file

this is particularly useful when you're setting config through the env vars. Could we use that here as well?